.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Description of Changes
+---
+Replace this text with an overview of your changes made in this Pull Request. Please use your best judgement here, do not be verbose to the point that you are giving an exact step-by-step of your workflow, but do not undersell the changes made. If this Pull Request addresses an open issue, you should reference that too.
+
+### Visual Sample
+---
+Replace this text with a media attachment or code test snippet that demonstrates the functionality of your changes. Please be mindful that GitHub is a platform for everyone, refrain from sending big attachments that could take a very long time to download for users with slow internet speeds.
+
+### Checklist
+---
+
+- [ ] I have read the LibreQuake contribution guidelines
+- [ ] I have thoroughly tested my changes to the best of my ability
```

Comment:
This one is kind of meh. Yeah sure but the Visual Sample thing kind of takes care of this already (almost) and this is just a reminder that I have to check everytime but what does it really mean then. WIll I go "Ah, you know lets stop right here and do some testing before I open this PR". I kinda doubt it.

Reply:
My thought process when opting to add this was a prediction that many contributors will see something work in their very specific case (such as a test map) and assume things are working correctly. I wanted the act of checking this box to be a conscious decision, and acknowledgement that we take finding regressions seriously.

I agree if you are like me (for better or for worse) you will go "this check is a waste of my time" but considering the fact we have a lot of artists who come-in-and-get-out, we might actually be able to encourage them to spend a little more time verifying their changes.

Curious about your thoughts, happy to revise if you disagree.

Reply:
We can leave it in. Also I didn't know that you can click to check the boxes so that actually becomes less tedious.